website/blog/2023-09-22-query-filter-ferretdb-with-majorm.md

```diff
@@ -0,0 +1,165 @@
+
+With FerretDB, you're never short of MongoDB GUI options to choose from, and in this blog post, we'll be diving into one of them: MajorM.
+
+<!--truncate-->
+
+Often, with proprietary software solutions like MongoDB, many find themselves constrained in terms of flexibility and costs and are always at [risk of vendor lock-in](https://blog.ferretdb.io/5-ways-to-avoid-database-vendor-lock-in/).
+That's why _truly_ open-source software like FerretDB is so important and attractive!
+
+FerretDB won't constrict you or lock you in.
+What's even better, you get to use many of your favorite MongoDB tools with FerretDB.
+Because of that, you can easily leverage a MongoDB GUI like MajorM for FerretDB, just as you would for MongoDB.
+
+MajorM is a simple MongoDB GUI desktop app that provides fast and complex query searches, visual query builder, and drag-and-drop functionality.
+It also provides a `mongoshell` UI for advanced operations on your FerretDB instance.
+
+This guide is just one of a series of blog posts focusing on using MongoDB GUI tools to query and filter FerretDB instances.
+See other blog posts below:
+
+- [Using FerretDB 1.0 with Studio 3T](https://blog.ferretdb.io/using-ferretdb-with-studio-3t/)
+- [Using Mingo to Analyze and Visualize FerretDB Data](https://blog.ferretdb.io/using-mingo-analyze-visualize-ferretdb-data/)
+- [MongoDB GUI: Using FerretDB with NoSQLBooster](https://blog.ferretdb.io/mongodb-gui-using-ferretdb-nosqlbooster/)
+
+## Prerequisites
+
+Before we get started, ensure to have the following set up before proceeding to other parts of the blog post.
+
+- [Download the MajorM desktop application from the website](https://www.majorm.app/)
+- [Install Docker](https://docs.docker.com/get-docker/) to set up the FerretDB instance
+- mongosh
+
+## Set up FerretDB
+
+Follow this Docker installation guide to launch an instance of FerretDB and have your connection string ready.
+
+For this tutorial, we'll be setting up our docker-compose file with the following configurations.
+
+```yaml
+services:
+  postgres:
+    image: postgres
+    environment:
+      - POSTGRES_USER=username
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_DB=ferretdb
+    volumes:
+      - ./data:/var/lib/postgresql/data
+  ferretdb:
+    image: ghcr.io/ferretdb/ferretdb
+    restart: on-failure
+    ports:
+      - 27017:27017
+    environment:
+      - FERRETDB_POSTGRESQL_URL=postgres://postgres:5432/ferretdb
+networks:
+  default:
+    name: ferretdb
+```
+
+Here, we've set up the `ferretdb` service to connect to the `postgres` service, as it translates MongoDB commands into SQL for the underlying PostgreSQL backend.
+Then we'll launch the services with `run docker compose up -d`.
+
+### Set up the MajorM connection to FerretDB
+
+Launch your installed MajorM application.
+
+Click "New Connection" to connect to FerretDB.
+
+![New Connection section](/img/blog/ferretdb-majorm/1-connections-tab.png)
```

Comment:
https://github.com/FerretDB/FerretDB/blob/dabbeb33218eee2f288f4cfa7cd7c52dc0d33204/website/docs/contributing/writing-guide.md?plain=1#L71

So either that link or our guide is not correct

Reply:
@AlekSi PTAL https://github.com/FerretDB/FerretDB/pull/3424